src/Sulu/Bundle/AdminBundle/Resources/js/components/RectangleSelection/tests/ModifiableRectangle.test.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable flowtype/require-valid-file-annotation */
+import React from 'react';
+import {shallow, mount} from 'enzyme';
+import ModifiableRectangle from '../ModifiableRectangle';
+
+test('The component should render', () => {
+    const view = mount(<ModifiableRectangle width={200} height={100} />);
+
+    expect(view).toMatchSnapshot();
+});
+
+test('The component should render with correct positions', () => {
+    const view = mount(<ModifiableRectangle width={200} height={100} left={10} top={20} />);
+
+    expect(view).toMatchSnapshot();
+});
+
+test('The component should call the double click callback', () => {
+    const clickSpy = jest.fn();
+    const rectangle = shallow(<ModifiableRectangle onDoubleClick={clickSpy} width={200} height={100} />);
+
+    rectangle.simulate('dblclick');
+    expect(clickSpy).toHaveBeenCalledTimes(1);
+});
+
+test('The component should call the change callback on move', () => {
+    const bodyListeners = {};
+    const changeSpy = jest.fn();
+    document.body.addEventListener = jest.fn((event, cb) => bodyListeners[event] = cb);
+
+    const rectangle = mount(<ModifiableRectangle onChange={changeSpy} width={200} height={100} />);
+    expect(bodyListeners.mousemove).toBeDefined();
+    expect(bodyListeners.mouseup).toBeDefined();
+
+    rectangle.simulate('mousedown', {pageX: 10, pageY: 20});
+    bodyListeners.mousemove({pageX: 15, pageY: 30});
+
+    expect(changeSpy).toHaveBeenCalledTimes(1);
+    expect(changeSpy).toHaveBeenCalledWith({topChange: 10, leftChange: 5, widthChange: 0, heightChange: 0});
+
+    bodyListeners.mouseup();
+    bodyListeners.mousemove({pageX: 100, pageY: 200});
+
+    expect(changeSpy).toHaveBeenCalledTimes(1);
+});
+
+test('The component should call the change callback on resize', () => {
+    const bodyListeners = {};
+    const changeSpy = jest.fn();
+    document.body.addEventListener = jest.fn((event, cb) => bodyListeners[event] = cb);
+
+    const rectangle = mount(<ModifiableRectangle onChange={changeSpy} width={200} height={100} />);
+    const resizeHandle = rectangle.find('.resizeHandle');
+    expect(bodyListeners.mousemove).toBeDefined();
+    expect(bodyListeners.mouseup).toBeDefined();
+
+    resizeHandle.simulate('mousedown', {pageX: 10, pageY: 20});
+    bodyListeners.mousemove({pageX: 15, pageY: 30});
+
+    expect(changeSpy).toHaveBeenCalledTimes(1);
+    expect(changeSpy).toHaveBeenCalledWith({topChange: 0, leftChange: 0, widthChange: 5, heightChange: 10});
+
+    bodyListeners.mouseup();
+    bodyListeners.mousemove({pageX: 100, pageY: 200});
+
+    expect(changeSpy).toHaveBeenCalledTimes(1);
```

Comment:
The behaviour that i want to test is, that after a `mouseup` a `mousmove` does not have any effects.